kolibri/plugins/coach/assets/src/app.js

```diff
@@ -88,10 +88,11 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.RECENT_LEARNER_ITEM_DETAILS,
-          path: '/:classId/recent/:channelId/:contentId/:userId',
+          path: '/:classId/recent-learner-item-details/:userId/:channelId/item/:contentId/:questionNumber?/:interactionIndex?',
```

Comment:
I was expecting that we'd keep the URL pattern consistent with the other coach URLs. E.g.

```
/:classId/recent/:channelId/:contentId/:userId/:questionNumber?/:interactionIndex?
```

particular reason that it was changed, or was it just copied from the `EXERCISE_RENDER` pattern?